src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    setIsLoading(true);
+    setIsError(false);
+    getPeople()
+      .then(setPeople)
+      .catch(() => setIsError(true))
+      .finally(() => setIsLoading(false));
```

Comment:
In the current implementation, if an error occurs during the fetching of data, the error state is set to true, but the loading state is still set to false. This means that if an error occurs, the loading spinner will disappear, but the error message will not be shown because the loading state is still true. To fix this, you should set the loading state to false in the catch block as well. This way, if an error occurs, the loading state will be set to false and the error message will be displayed.